src/views/Login.vue

```diff
@@ -122,6 +123,12 @@ export default defineComponent({
         return
       }
 
+      await this.fetchLoginOptions()
+      // show OMS input if SAML if configured
+      if (this.loginOption.loginAuthType !== 'BASIC') {
```

Comment:
We can move this check with this one https://github.com/hotwax/launchpad/pull/45/files#diff-08aaad4ec5901f788c2efff31ebfa22a4132abc8048ee9685469a1ef570f0876L131

Reply:
Updated sir